src/main/java/Dude.java

```diff
@@ -0,0 +1,213 @@
+import java.io.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Dude {
+
+    static ArrayList<Task> taskList = new ArrayList<Task>();
+    static int nTasks = 0;
+
+    private static final String FILE_PATH = "./data/dude.txt";
```

Comment:
This constant doesn't seem to be used. Did you forget to replace your instances of `"./data/dude.txt"` with `FILE_PATH`?